src/specklepy/objects/other.py

```diff
@@ -303,9 +303,19 @@ class RevitParameter(Base, speckle_type="Objects.BuiltElements.Revit.Parameter")
     isTypeParameter: bool = False
 
 
+@deprecated(version="2.20", reason="Namespace changed, collectionType deprecated")
 class Collection(
     Base, speckle_type="Speckle.Core.Models.Collection", detachable={"elements"}
 ):
     name: Optional[str] = None
     collectionType: Optional[str] = None
     elements: Optional[List[Base]] = None
+
+
+class Collection(  # noqa: F811
```

Comment:
Will two classes with the same name work? is that what this F811 suppression is for?
I'm a bit worried that this may break the Blender connector...

Reply:
@JR-Morgan oops didn't see your review 🙈
yep, I tested, and it is the same way it works with VectorLayer for example. But I am open to any other possibility, it's just I haven't found a way to safely change one speckle_type to another